examples/application-composer/README_pt.adoc

```diff
@@ -0,0 +1,164 @@
+= Compositor de Aplicações
+:index-group: Testing Techniques
+:jbake-type: page
+:jbake-status: published
+
+O `org.apache.openejb.junit.ApplicationComposer` é o corredor de testes JUnit modelado de acordo com a forma como fizemos testes internos no OpenEJB por anos (desde 2006).
```

Comment:
Hi Daniel, in the sentence "corredor de testes" i believe that "executor de testes" make more sense